x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/palette_indicator.tsx

```diff
@@ -5,23 +5,14 @@
  */
 
 import React from 'react';
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiColorPaletteDisplay } from '@elastic/eui';
 import { AccessorConfig } from '../../../types';
 
 export function PaletteIndicator({ accessorConfig }: { accessorConfig: AccessorConfig }) {
   if (accessorConfig.triggerIcon !== 'colorBy' || !accessorConfig.palette) return null;
   return (
-    <EuiFlexGroup className="lnsLayerPanel__paletteContainer" gutterSize="none" alignItems="center">
-      {accessorConfig.palette.map((color) => (
-        <EuiFlexItem
-          key={color}
-          className="lnsLayerPanel__paletteColor"
-          grow={true}
-          style={{
-            backgroundColor: color,
-          }}
-        />
-      ))}
-    </EuiFlexGroup>
+    <div className="lnsLayerPanel__paletteContainer">
+      <EuiColorPaletteDisplay size="xs" palette={accessorConfig.palette} />
```

Comment:
Thanks for making this change, @flash1293. Super minor nitpick, but any chance we can do the following to apply two small style overrides to this `EuiColorPaletteDisplay` (to make it look more at home in the dimension item)?

1. Apply a `className="lnsLayerPanel__palette"` to `EuiColorPaletteDisplay`.
2. Apply the following styles to this class to adjust the border radius and remove the border:

```
.lnsLayerPanel__palette {
  border-radius: 0 0 ($euiBorderRadius - 1px) ($euiBorderRadius - 1px);

  &::after {
    border: none;
  }
}
```